vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/Grid.java

```diff
@@ -3295,4 +3349,225 @@ private void handleDataProviderChange(DataProvider<T, ?> dataProvider) {
         dataProviderChangeRegistration = dataProvider
                 .addDataProviderListener(event -> onDataProviderChange());
     }
+
+    /**
+     * Adds a drop listener to this component.
+     *
+     * @param listener
+     *            the listener to add, not <code>null</code>
+     * @return a handle that can be used for removing the listener
+     */
+    @SuppressWarnings({ "unchecked", "rawtypes" })
+    public Registration addDropListener(
```

Comment:
Are you sure that these method names won't conflict the generic DnD feature? Probably those would not be useful for Grid anyway and there's no problem here

Reply:
Good point. @pleku should these APIs be changed to `addGridDropListener` or `addRowDropListener` etc to avoid conflicts?